File: level1/가운데_글자_가져오기/Solution.java
class Solution {
    public String solution(String s) {
        int c = s.length() / 2;
        String answer = s.substring(c - (s.length() % 2 == 0 ? 1 : 0), c + 1);
        return answer;
    }

    public static void main(String[] args) {
        Solution s = new Solution();
        System.out.println(s.solution("abcde"));
        System.out.println(s.solution("qwer"));
    }
}

File: level1/가운데_글자_가져오기/solution.js
function solution(s) {
  var even = s.length % 2 === 0 ? 1 : 0;
  var answer = s.substr(s.length / 2 - even, 1 + even);
  return answer;
}

console.log(solution("abcde"));
console.log(solution("qwer"));
